OpenRA.Game/Sound/Sound.cs

```diff
@@ -364,30 +364,30 @@ public static float VideoSeekPosition
 			return true;
 		}
 
-		public static bool PlayVoice(string phrase, Actor voicedUnit, string variant)
+		public static bool PlayVoice(string phrase, Actor voicedActor, string variant)
```

Comment:
It would be great if we could move this into the IVoiced trait implementations themselves.  The sound code really shouldn't be digging around inside actors.